docs/accessibility.md

```diff
@@ -0,0 +1,221 @@
+# Accessibility
+
+## General Practices
+
+As a general rule, accessible software is achieved by following web development best practices. As Treeherder uses the `reactstrap` library, it is recommended using one of its components, if possible.
+
+If you can not find a suitable component, prefer [HTML5 semantic tags](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#Semantics_in_HTML) and avoid using generic tags, such as `span` and `div`. Use those only when there is no other more semantic tag. Moreover, try to use more **headings** (`h1` to `h6`) whenever possible, because screen readers users usually navigate by headings and that is only possible when developers insert them in the code.
+
```

Comment:
I would omit this sentence " Use those only when there is no other more semantic tag" would be clearer as " and add to the last sentence "because it is not accessible to screen readers" (or however you want to word that).